modules/iam-user-for-devs/variables.tf

```diff
@@ -0,0 +1,6 @@
+variable "users" {
+    description = "Set of user names"
+
+    type = set(string)
+    default = ["testuser"]
```

Comment:
default does not make sense here, it should be deleted, users variable should be required and always provided, otherwise the whole module does nothing